src/components/TokenSelect/TokenButton.tsx

```diff
@@ -9,10 +9,10 @@ import Row from '../Row'
 import TokenImg from '../TokenImg'
 
 const StyledTokenButton = styled(Button)<{ approved?: boolean }>`
-  border-radius: ${({ theme }) => theme.borderRadius}em;
-  min-height: 2.25em;
+  border-radius: 1em;
```

Comment:
you can define this in terms of `theme.borderRadius.medium` once that PR goes in :P